src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    protected static String[] readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        List<String> statistic = null;
+        try {
+            statistic = Files.readAllLines(file.toPath());
```

Comment:
let's read line by line and process each line instead of assembling all line together, then splitting them once more.

Reply:
done